servicecontrol/external-integrations.md

```diff
@@ -23,9 +22,13 @@ The notifications contain information about following aspects:
  * The message itself (e.g. the headers and, if using non-binary serialization, also the body)
 
 
-### Subscribing for failed message notifications
+### How do I subscribe to notifications
+
+We recommend that you use a separate endpoint from your business endpoints to handle the integration events.
 
-Assuming an existing NServiceBus endpoint is to be used, these steps need to be done in order to get the failed message notifications from ServiceControl
+NOTE: Make sure to use `error` and `audit` queues for this endpoint that isn't monitored by ServiceControl. If not search result might include integration events and failures in the integration endpoint might cause infinite loops since they will trigger new message failed events.
```

Comment:
I'll do a second pass on the whole thing and ping you, just dumped in some
text to not forget. (Hence the `[WIP]`  :)  )

On Fri, Nov 13, 2015 at 2:31 PM, Simon Cropp <notifications@github.com>
wrote:

> In servicecontrol/external-integrations.md
> <https://github.com/Particular/docs.particular.net/pull/920#discussion_r44781867>
> :
>
> >
> > -Assuming an existing NServiceBus endpoint is to be used, these steps need to be done in order to get the failed message notifications from ServiceControl
> > +NOTE: Make sure to use `error` and `audit` queues for this endpoint that isn't monitored by ServiceControl. If not search result might include integration events and failures in the integration endpoint might cause infinite loops since they will trigger new message failed events.
>
> If not search result might include integration events and failures in the
> integration endpoint might cause infinite loops since they will trigger new
> message failed events.
>
> please break up
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/docs.particular.net/pull/920/files#r44781867>
> .
>